src/policy/feerate.cpp

```diff
@@ -7,29 +7,26 @@
 
 #include <tinyformat.h>
 
-CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nBytes_)
+CFeeRate::CFeeRate(const CAmount& nFeePaid, uint32_t num_bytes)
```

Comment:
nit

```cpp
CFeeRate::CFeeRate(const CAmount& nFeePaid, uint32_t num_bytes)
    : nSatoshisPerK(num_bytes > 0 ? nFeePaid * 1000 / int64_t{num_bytes} : 0)
{
}
```
And make `const CAmount nSatoshisPerK`.

Reply:
Nice, but adding `const` can be done in a follow-up